examples/playwright/package.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "@theia/playwright-testframework",
```

Can likely be simplified to:

```suggestion
  "name": "@theia/playwright",
```